src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -19,13 +20,77 @@ type HacbsApplicationDetailsProps = {
 const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicationName }) => {
   const { namespace } = React.useContext(NamespaceContext);
   const navigate = useNavigate();
+  const { quickStarts } = useChrome();
+
   const [application, applicationsLoaded] = useK8sWatchResource<ApplicationKind>({
     groupVersionKind: ApplicationGroupVersionKind,
     name: applicationName,
     namespace,
   });
   const appDisplayName = application?.spec?.displayName || '';
 
+  useEffect(() => {
+    const hacbsShowAppQs = localStorage.getItem('hacbsShowAppQs');
+    if (hacbsShowAppQs === null) {
```

Comment:
Check against the value that you set instead of the presence of _any_ value.
```suggestion
    if (hacbsShowAppQs !== 'false') {
```

Reply:
> Check against the value that you set instead of the presence of _any_ value.

My thinking was that we want to check for a null key because if it's run once, then the existence of the key means it has been run. I thought we'd reserve checking for ! false if we decide to go with an 'Always show on completion' option.